analysis/constants.py

```diff
@@ -0,0 +1,5 @@
+PANDAS_CHUNKSIZE = 1000
```

Comment:
To support parameterization without having to touch the code it should read config values from environment variables, falling back to default values if not present.

```suggestion
int(os.getenv("MANTA_PANDAS_CHUNKSIZE", "1000"))
```